src/sentry/auth/superuser.py

```diff
@@ -275,12 +282,39 @@ def set_logged_in(self, user, current_datetime=None):
         request = self.request
         if current_datetime is None:
             current_datetime = timezone.now()
+
+        token = get_random_string(12)
+
+        try:
+            su_access_json = json.loads(request.body)
+        except json.JSONDecodeError:
+            su_access_json = []
+
+        if "superuserAccessCategory" in su_access_json:
+
+            su_access_info = SuperuserAccessSerializer(data=su_access_json)
+
+            if not su_access_info.is_valid():
+                raise serializers.ValidationError(su_access_info.errors)
+
+            logger.info(
+                "superuser.superuser_access",
+                extra={
+                    "superuser_session_id": token,
```

Comment:
let's rename the var `token` to `superuser_session_id`

Reply:
do we want to change all instances of `token` into `superuser_session_id` or just within this method?

Reply:
if its used in a lot of places, then makes sense to log it as `superuser_token_id`